src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -343,7 +375,6 @@
     <PropertyGroup Condition="'$(SkipVersionGeneration)'!='true'">
       <BuildNumberMajor >$(GeneratedBuildNumberMajor)</BuildNumberMajor>
```

Comment:
Nevermind, I see it here: https://github.com/alexperovich/buildtools/blob/daac863e9ea367338d6295c42264a58f4eb5bf1e/src/Microsoft.DotNet.Build.Tasks/PackageFiles/BuildVersion.targets